L59-core-allow-cppstdlib.md

```diff
@@ -0,0 +1,122 @@
+Allow C++ standard library in gRPC Core Library
+----
+* Author(s): veblush
+* Approver: a11r
+* Status: Draft
+* Implemented in: n/a
+* Last updated: August 19, 2019
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/umQyyXmNr2w
+
+## Abstract
+
+Allow C++ Standard Library to be used in the gRPC Core library.
+
+## Background
+
+Since C++ was allowed to write gRPC, more and more code has been written in C++.
+But most features from C++ standard libraries were not allowed to use because
+we didn't want to introduce new dependency against C++ standard library.
+
+The decision to use C++ has been helping us to be more productive but it's not
+best because we cannot use C++ standard library and a few of C++ features which
+depend on this. This also prevents us from using other C++ library which rely 
+on the library. (e.g. abseil)
+
+This gRFC proposes allowing C++ standard library to be used within the gRPC 
+Core library. It may free us from reinventing wheels which already exists
+in the standard library and help us become more productive.
+
+### Related Proposals:
+
+* [Allow C++ in gRPC Core Library](L6-core-allow-cpp.md)
+
+## Proposal
+
+Allow C++11 standard library usage in gRPC Core. The previous C++ usage rule
+will be valid except the restriction on C++ features and header-only library.
+
+- All C++11 features are allowed to use. Followings were not possible to use
```

Comment:
rephrase as:
"Following C++11 features will become available with this proposal"?

Reply:
Done.